parquet-column/src/main/java/org/apache/parquet/column/values/bloomfilter/BlockSplitBloomFilter.java

```diff
@@ -394,4 +395,21 @@ public long hash(float value) {
   public long hash(Binary value) {
     return hashFunction.hashBytes(value.getBytes());
   }
+
+  @Override
+  public void putAll(BloomFilter otherBloomFilter) throws IOException {
```

Comment:
Could you add some tests to verify if the merged BF is as expected?

Reply:
> Could you add some tests to verify if the merged BF is as expected?

Sure, I will add some UT later